charts/cloudserver-front/templates/dashboard.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "cloudserver-front.fullname" . }}-grafana-dashboard
+  labels:
+    app: {{ template "cloudserver-front.name" . }}
+    chart: {{ template "cloudserver-front.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    grafana-dashboard: "true"
+data:
+  cloudserver-front.json: |
+{{ .Files.Get "dashboards/cloudserver-front.json" | indent 4 }}
```

Comment:
Use `.Files.Glob` instead https://github.com/kubernetes/helm/blob/master/docs/chart_template_guide/accessing_files.md#configmap-and-secrets-utility-functions

Reply:
Done!